osu.Desktop/Linux/.gitignore

```diff
@@ -0,0 +1 @@
+!*.nupkg
```

Comment:
If we're adding this, it should be in the global `.gitignore`. Not sure it's required (if this is bothering you then you can set an ignore in your home directory gitignore file.

Reply:
I added this along the package for this branch. Both should be removed eventually.

Reply:
I removed the gitignore since it does not have any purpose.